addons/swap-local-global/addon.json

```
@@ -1,5 +1,5 @@
 {
-  "name": "Switch variables between \"For all sprites\" and \"For this sprite only\"",
+  "name": "Switch variables between all sprites and this sprite",
```

This one's hard to improve too, but I have some ideas:
```suggestion
  "name": "Switch variables between for all sprites and this sprite",
```
```suggestion
  "name": "Switch variables between local and global",
```
```suggestion
  "name": "Variable type switching",
```

---

Do most Scratchers know the term scope to use that?

---

> Do enough Scratchers know the term scope to just use that?

I don't think "scope" is a familiar term in the Scratch community. Also, it looks like this addon can change whether variables are Cloud variables or not, and "variable _type_ switching" would fit that.